_rules/block-collapsible-3e12e1.md

```diff
@@ -54,7 +54,9 @@ Note that if there is no [block of repeated content][] before the [main block of
 
 - [Technique SCR28: Using an expandable and collapsible menu to bypass block of content][tech scr28]
 
-In the test cases, the `aside` and `nav` elements are each a [block of repeated content][] due to the link inside the `nav` element to a page with similar [blocks of content][block of content]; and the `main` element is the [main block of content][].
+In order to focus on only on the part of the associated composite rule ([_Bypass blocks of content_][bypass blocks]) which this atomic rule illustrate, and given the very nature of some of the other input rules, test cases use bold text instead of heading (to avoid also passing rule [_Document has heading for main section of content_][document has heading for main]) and a `<div id="main">` instead of a `main` element (to avoid also passing rule [_Document has a main landmark_][document has main]). These are bad practices and should be avoided. In addition, this often create examples that do not satisfy [Success Criterion 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships).
```

Comment:
Should this be mentioned in the assumptions? Would placing this right underneath the "test cases" header not be more appropriate?

Ditto with all the other such assumptions.

Reply:
These are in the Background section, not in the Assumptions. This is indeed not an Assumption but some high-level explanation valid for all examples.

I do agree that it could also fit under the "Test Cases" headings. Or that we could structure Background a bit better.
We've only started using Background fairly recently (when killing out notes). My current stance is to have first the background about the rule, then the list of links ("bibliography", to some extend), and then the bakground about the test cases. This is likely not optimal…

I'm all up for having a CG discussion on what to do with these extended backgrounds we now have, whether it's sub-headings, move some bit under the Test Cases heading, … I think we start to have enough examples of what we put in Background to start having an idea of what can be done. This is however a separate topic and I suggest keeping it here in this PR and revisiting if we decide to change.

Reply:
Oh right. Silly GitHub confused me. I am fine with this being in the background. Thank you for the explanation! I will approve for now but if this is moved to the test cases or if backgrounds are restructured better I am fine with that too.